src/utilities/models.py

```diff
@@ -80,6 +80,7 @@ class Meta:
 
     def __str__(self):
         target = 'target="_blank"' if self.open_link_in_new_tab else ''
-        return '<a href="{}" {}>' \
+        # append dummy (.menuitem) CSS class, fix #1397
```

Comment:
Can remove this.  Issue #'s don't link and can be ambiguous, so if you want to include issues for contenct, paste the entire url not just a #.  In this case, comment is unnecessary.  It is more of a PR comment that is helpful for review, but doesn't add much info to the code base for future devs.

Reply:
corrected, see latest commit https://github.com/bytedeck/bytedeck/pull/1456/commits/b7a28dae9617ba7b29d20c5b5ac9ef58ead8d249